client/src/components/RecipeForm/RecipeForm.js

```diff
@@ -203,56 +219,60 @@ class RecipeForm extends Component {
                     <Col md={4} className="specs">
                         <Row>
                             <Col md={6}>
-                                <FormGroup>
+                                <FormGroup validationState={this.numMinMaxValidation(this.state.batchVol, 0, 10000)}>
                                     <ControlLabel className="formLabels">Batch Volume:</ControlLabel>
                                     <InputGroup>
-                                        <FormControl type='number' name='batchVol' value={this.state.batchVol} onChange={this.handleInputChange} id='batchVol' placeholder='Batch'/>
+                                        <FormControl type='number' name='batchVol' min={1} max={10000} value={this.state.batchVol} onChange={this.handleInputChange} id='batchVol' placeholder='Batch'/>
```

Comment:
Split this into separate lines:
<FormControl
         type='number'
         name='batchVol'
         ... />